### PR TITLE
HPCC-28638 Base Build Image for ubuntu-22.04

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -320,3 +321,7 @@ prefab/
 pythonenv3.8/
 .venv/
 
+###################
+# hpcc
+###################
+.env
```
Comment (Michael-Gardner) [inline .gitignore:327]: Is this file being used locally for the build.sh script?  I see it ignored here but exported there.

**Review by Michael-Gardner — APPROVED** [review]
Tested ubuntu-22.04.dockerfile locally.
Tested buildall.sh script. No issues.

File: .gitignore (hunk 1 above)
Comment (GordonSmith) [reply .gitignore:327]: Yes - its simply storing the secrets for NuGet and DockerHub (which can also be supplied as args to the script)

Comment (GordonSmith) [reply .gitignore:327]:
```
GITHUB_ACTOR=XXX
GITHUB_TOKEN=YYY
DOCKER_USERNAME=AAA
DOCKER_PASSWORD=BBB
```

**Pull request merged**